doc/languages-frameworks/python.md

```diff
@@ -804,6 +804,43 @@ If you want to create a Python environment for development, then the recommended
 method is to use `nix-shell`, either with or without the `python.buildEnv`
 function.
 
+### How to consume python modules using pip in a virtualenv like I am used to on other Operating Systems ?
+
+This is an example of a `default.nix` for a `nix-shell`, which allows to consume a `virtualenv` environment,
+and install python modules through `pip` the traditional way.
+
+Create this `default.nix` file, together with a `requirements.txt` and simply execute `nix-shell`.
+
+```
+
+th import <nixpkgs> {};
+with pkgs.python27Packages;
+
+buildPythonPackage { 
```

Comment:
we're not building a Python package, but an environment. I suggest you use `python.buildEnv` or else I think you might as well use `stdenv.mkDerivation`.